.github/workflows/e2e-nvidia-l4-x1.yml

```diff
@@ -73,7 +73,7 @@ jobs:
           mode: start
           github-token: ${{ secrets.GH_PERSONAL_ACCESS_TOKEN }}
           ec2-image-id: ${{ vars.AWS_EC2_AMI }}
-          ec2-instance-type: g6.8xlarge
+          ec2-instance-type: g6.16xlarge
```

Comment:
at the very least put a comment to revert it later; and link to https://github.com/instructlab/instructlab/issues/3298

That said, I don't think we should use bumps without first trying to collect some disk usage data on the node. I doubt the size of the disk is the actual problem with gguf writer. Having some monitoring / stats collection during job run that we could then dump to github logs and inspect would help here.